### ISSUE TITLE
Move Task status dropdown location on Task screen

### ISSUE THREAD

**Issue by carye3** [newjersey/business.nj.gov#466]
[Design](https://www.figma.com/file/Hg3ggwTj50wxdg4mjG4xRB/Dashboard-Exploration?node-id=4973%3A118076)

![image](https://user-images.githubusercontent.com/62653834/133844559-01877fed-7cce-4ad3-9b5d-42d6e6bb34e8.png)

**Comment by skeeJay** [comment]
@carye3 will let you QA from a design perspective. One think I noticed as a regression: when the dropdown opens, the location of the current status "jumps" to the right because the dropdown that appears is no longer "perfectly" aligned with the the button that opens it. I don't know if we care about that, but it's something that I think we implemented in the old location.

**Comment by carye3** [comment]
I only see a jump in the mini roadmap when status is changed to "complete"

<img width="311" alt="Screen Shot 2021-09-27 at 3 13 59 PM" src="https://user-images.githubusercontent.com/62653834/134970501-7295f1f2-2c4a-4217-8ca5-b38f7f2c0c7d.png">

<img width="330" alt="Screen Shot 2021-09-27 at 3 14 23 PM" src="https://user-images.githubusercontent.com/62653834/134970553-a6394438-0e1d-4285-88e5-14cf691bc57e.png">

**Comment by skeeJay** [comment]
@carye3 I think we're talking about different things. I am referring to the dropdown that was moved as part of this user story. The status text when the dropdown is closed no longer perfectly aligns with the status text when the dropdown is open. Does that make sense?

**Comment by skeeJay** [comment]
@carye3 Here's an illustration. In the previous iteration, Guy had tried to make sure when the dropdown opens, the dropdown forms "around" the current status text without it moving.

![Screen Shot 2021-09-27 at 3 36 35 PM](https://user-images.githubusercontent.com/506096/134973877-05408ca2-025a-402c-a995-0829da037bb2.png)

**Comment by skeeJay** [comment]
@FarazA22 In the old location, the dropdown appeared on top of the trigger button, so that the previously-selected option appeared in the flow of the dropdown and the other options "appeared" around it. In the new location, it shows all the options, including the previously-selected one. Is there any way to maintain the old behavior in the new location? Before and after below:

<img width="238" alt="Screen Shot 2021-09-29 at 3 02 57 PM" src="https://user-images.githubusercontent.com/506096/135332516-36d6220d-3fff-4f54-a1ac-2e88aa15b2fb.png">

<img width="279" alt="Screen Shot 2021-09-29 at 3 03 04 PM" src="https://user-images.githubusercontent.com/506096/135332525-e21bea45-65e7-4d09-97c5-922f8af7bb52.png">

**Comment by FarazA22** [comment]
@skeeJay @carye3 We've recently upgraded to Material UI v.5, with this upgrade, we are able to quickly change the position of the menu relative to the button, but the difference in v5 is that its position is fixed. In the previous example (MUI v4), the menu jumped around based on where the menu list item fell within the menu. Do we really want this behavior? I've presented two options we can quickly implement. I personally like option 2.



Menu closed
<img width="195" alt="Screen Shot 2021-09-29 at 4 26 54 PM" src="https://user-images.githubusercontent.com/72581979/135345566-14767361-e84d-44a7-8ee8-ac003956ae04.png">

Option 1 - centered relative to the button (AnchorOrigin / TransformOrigin are both V-Center, H -Center)
<img width="185" alt="Screen Shot 2021-09-29 at 4 26 58 PM" src="https://user-images.githubusercontent.com/72581979/135345581-2e7d3bf0-84ee-44af-be2a-0a859fa5d624.png">

Option 2 - position is from the top left of the button (AnchorOrigin / TransformOrigin are both V-Top, H -Left)
<img width="184" alt="Screen Shot 2021-09-29 at 4 28 57 PM" src="https://user-images.githubusercontent.com/72581979/135345582-a39f7b46-f363-401d-994e-f998776a60b3.png">

<img width="174" alt="Screen Shot 2021-09-29 at 4 28 50 PM" src="https://user-images.githubusercontent.com/72581979/135345584-c28241ed-a406-4c07-9371-1942a1c4610c.png">
